src/sentry/integrations/metric_alerts.py

```diff
@@ -96,3 +105,132 @@ def incident_attachment_info(incident, new_status: IncidentStatus, metric_value=
         "ts": ts,
         "title_link": title_link,
     }
+
+
+def metric_alert_attachment_info(
+    alert_rule: AlertRule,
+    selected_incident: Optional[Incident] = None,
+    new_status: IncidentStatus = None,
+    metric_value=None,
+    unfurl=False,
+):
+    logo_url = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
```

Comment:
Since this is shared between the two functions can you take it out as a constant?
```py
LOGO_URL = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
```